csv/Example_ANN_Training_Samples.csv

```diff
@@ -0,0 +1,830 @@
+0,1,0,0,0,Acne

```

Comment:
@alwinbaby could you confirm this was the dataset you used for training?

Reply:
Yup this is the one i used